1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/file/configs/AccountsConfig.java

```diff
@@ -16,7 +17,7 @@
 
 public class AccountsConfig extends FileConfig {
 
-    public final List<MinecraftAccount> altManagerMinecraftAccounts = new ArrayList<>();
+    private final List<MinecraftAccount> altManagerMinecraftAccounts = new ArrayList<>();
```

Comment:
This field shouldn't be called `altManagerMinecraftAccounts`. Please refactor it (and it's getters) to `accounts`